primitives/chain-crab-parachain/src/lib.rs

```diff
@@ -1,122 +0,0 @@
-// This file is part of Darwinia.
-//
-// Copyright (C) 2018-2022 Darwinia Network
-// SPDX-License-Identifier: GPL-3.0
-//
-// Darwinia is free software: you can redistribute it and/or modify
-// it under the terms of the GNU General Public License as published by
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-//
-// Darwinia is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-// GNU General Public License for more details.
-//
-// You should have received a copy of the GNU General Public License
-// along with Darwinia. If not, see <https://www.gnu.org/licenses/>.
-
-#![cfg_attr(not(feature = "std"), no_std)]
-
-mod copy_paste_from_darwinia {
-	// --- darwinia-network ---
-	use bp_darwinia_core::*;
-	// --- paritytech ---
-	use sp_version::RuntimeVersion;
-
-	pub const VERSION: RuntimeVersion = RuntimeVersion {
-		spec_name: sp_runtime::create_runtime_str!("Crab Parachain"),
-		impl_name: sp_runtime::create_runtime_str!("Darwinia Crab Parachain"),
-		authoring_version: 1,
-		spec_version: 4,
-		impl_version: 1,
-		apis: sp_version::create_apis_vec![[]],
-		transaction_version: 1,
-		state_version: 0,
-	};
-
-	pub const EXISTENTIAL_DEPOSIT: Balance = 0;
-}
-pub use copy_paste_from_darwinia::*;
-
-pub use bp_darwinia_core::*;
-
-// --- paritytech ---
-use bp_messages::{LaneId, MessageDetails, MessageNonce, UnrewardedRelayersState};
-use frame_support::Parameter;
-use sp_std::prelude::*;
-
-/// CrabParachain Chain.
-pub type CrabParachain = DarwiniaLike;
-
-/// Name of the With-CrabParachain messages pallet instance that is deployed at bridged chains.
-pub const WITH_CRAB_PARACHAIN_MESSAGES_PALLET_NAME: &str = "BridgeCrabParachainMessages";
```

Comment:
It's used here: https://github.com/darwinia-network/darwinia-parachain/blob/main/runtime/crab-parachain/src/bridges_message/crab.rs#L72